### PR TITLE
Bumping versions

### PR DESCRIPTION
**Pull request by heschlie**
- Bumped libcalico-go
- Bumped confd
- Bumped felix

### CODE REVIEW

File: glide.yaml [hunk 1]
```diff
@@ -21,7 +21,7 @@ import:
   - json
 - package: github.com/projectcalico/go-yaml-wrapper
 - package: github.com/projectcalico/libcalico-go
-  version: 1.2.0
+  version: 1.2.1-rc1
```
Comment (gunjan5) [inline glide.yaml]: this should be `v1.2.1-rc1`, missing the `v` (glide might be smart enough to figure it out I guess, but we should make is explicit)

Comment (heschlie) [inline glide.yaml]: gah so close

Comment (robbrockbank) [inline glide.yaml]: Yeah I'm pretty sure glide does figure this out - I don't think we usually put the v's in but i' might be mistaken.

**Comment by gunjan5** [comment]
LGTM, squash and merge away!

**Pull request merged**